mne/io/eeglab/eeglab.py

```diff
@@ -1,12 +1,15 @@
-# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.com>
 #
 # License: BSD (3-clause)
 
+import warnings
```

Comment:
You still want to do `with warnings.catch_warnings(record=True)` in tests to catch any raised warnings, if that's what you're asking. Otherwise it's a drop-in, with a changed default UserWarning to RuntimeWarning.